.github/workflows/packaging.yml

```
@@ -0,0 +1,26 @@
+name: Packaging
+
+on:
+  release:
+    types: [released]
+
+
+jobs:
+  winget:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup winget-create
+        shell: powershell
+        run: |
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+      - name: Parse release version from github reference
+        run: Set-Item -PATH Env:\GITHUB_CLI_VERSION -Value ${{ github.ref }}.substring(1) 
+      - name: Update manifest
+        shell: powershell
+        run: |
+            wingetcreate.exe update --id GitHub,Inc.GitHubCLI --url https://github.com/cli/cli/releases/download/v$env:GITHUB_CLI_VERSION/gh_${env:GITHUB_CLI_VERSION}_windows_amd64.msi --version ${env:GITHUB_CLI_VERSION} --out .\manifest\
```

```suggestion
            wingetcreate.exe update --id GitHub.cli --url https://github.com/cli/cli/releases/download/v$env:GITHUB_CLI_VERSION/gh_${env:GITHUB_CLI_VERSION}_windows_amd64.msi --version ${env:GITHUB_CLI_VERSION} --out .\manifest\
```
Please don't create a duplicate ID for this in microsoft/winget-pkgs. We've already got it in the repo!

---

Thanks @jedieaston. I am happy to make that as a change, the only reason why I made it with this identifier is that is what Winget suggested with default, and was not sure if GitHub wanted a different Identifier naming for all their packages or not. If someone from their team can let me know, would be happy to make that change 😄

---

> Thanks @jedieaston. I am happy to make that as a change, the only reason why I made it with this identifier is that is what Winget suggested with default, and was not sure if GitHub wanted a different Identifier naming for all their packages or not. If someone from their team can let me know, would be happy to make that change 😄

`wingetcreate` is on Alpha Release 0.0.1 which have A TON of issues and was supposed to be for us Package Maintainers to test and easily update existing packages, `GitHub,Inc` we cannot allow, `GitHubInc` is possible but looks dumb, `GitHub` is the best looking non confusing way.

---

Sounds good to me. Approved the changes